src/bio.c

```diff
@@ -82,12 +73,31 @@ static unsigned int bio_job_to_worker[] = {
     [BIO_RDB_SAVE] = 3,
 };
 
-static pthread_t bio_threads[BIO_WORKER_NUM];
-static pthread_mutex_t bio_mutex[BIO_WORKER_NUM];
-static pthread_cond_t bio_newjob_cond[BIO_WORKER_NUM];
-static list *bio_jobs[BIO_WORKER_NUM];
+typedef struct {
+    const char *const bio_worker_title;
+    pthread_t bio_thread_id;
+    pthread_mutex_t bio_mutex;
+    pthread_cond_t bio_newjob_cond;
+    list *bio_jobs;
+} bio_worker_data;
+
+static bio_worker_data bio_workers[] = {
+    {"bio_close_file"},
+    {"bio_aof"},
+    {"bio_lazy_free"},
+    {"bio_rdb_save"},
+};
+static const bio_worker_data *const bio_worker_end = bio_workers + (sizeof bio_workers / sizeof *bio_workers);
+
+static size_t bioWorkerNum(const bio_worker_data *const bwd) {
+    // Ensure the pointer is valid - casting to uintptr_t to not make the comparison itself UB in case
+    // the pointer is outside the valid range. It's a best effort thing anyway.
```

Comment:
we usually use this
```suggestion
    /* Ensure the pointer is valid - casting to uintptr_t to not make the comparison itself UB in case
     * the pointer is outside the valid range. It's a best effort thing anyway. */
```

Reply:
> we usually use this

Oki! I changed that everywhere to make it consistent.

Reply:
@enjoy-binbin I've now extracted the `_Thread_local` and the fix for the commenting inconsistencies and put that into a separate patch.